docs/releases/5.0.2.txt

```diff
@@ -32,3 +32,8 @@ Bugfixes
 * Fixed a regression in Django 5.0 that caused the ``request_finished`` signal to
   sometimes not be fired when running Django through an ASGI server, resulting
   in potential resource leaks (:ticket:`35059`).
+
+* Fixed a bug in Django 5.0 that caused a crash when migrating models with a
+  literal ``db_default`` of a complex type such as ``dict`` instance of a
+  ``JSONField``. Migrations for fields using ``db_default`` might have to be
+  regenerated but the resulting SQL should be the same (:ticket:`35149`).
```

Comment:
Basically `makemigrations` will generate new migrations that make models defined as `Field(db_default=42)`  from `Field(db_default=Value(42))` to `Field(db_default=42)` but the first commit of this MR ensures this is a noop.